mmv1/products/networkservices/GrpcRoute.yaml

```diff
@@ -0,0 +1,247 @@
+    send_empty_value: true
+  - !ruby/object:Api::Type::Array
+    name: rules
+    description: |
+          Rules that define how traffic is routed and handled.  
+    required: true
+    send_empty_value: true
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties: 
+        - !ruby/object:Api::Type::Array
+          name: matches
+          description: |
+            Matches define conditions used for matching the rule against incoming gRPC requests.
+          send_empty_value: true
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::Array
+                name: headers
+                description: |
+                  Specifies a list of HTTP request headers to match against. 
+                send_empty_value: true
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: key
+                      description: |
+                        Required. The key of the header.
+                      required: true
+                    - !ruby/object:Api::Type::String
+                      name: value
+                      description: |
+                        Required. The value of the header.
+                      required: true
+                    - !ruby/object:Api::Type::Enum
+                      name: 'type'
+                      description: |
+                        The type of match.             
+                      values:
+                        - :TYPE_UNSPECIFIED
+                        - :EXACT
+                        - :REGULAR_EXPRESSION
+                      default_value: :EXACT    
+              - !ruby/object:Api::Type::NestedObject
+                name: method
+                description: |
+                  A gRPC method to match against. If this field is empty or omitted, will match all methods.
+                send_empty_value: true
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: grpcService
+                    description: |
+                      Required. Name of the service to match against. 
+                    required: true
+                  - !ruby/object:Api::Type::String
+                    name: grpcMethod
+                    description: |
+                      Required. Name of the method to match against. 
+                    required: true
+                  - !ruby/object:Api::Type::Boolean
+                    name: caseSensitive
+                    description: |
+                      Specifies that matches are case sensitive. The default value is true.
+                    default_value: true              
+        - !ruby/object:Api::Type::NestedObject
+          name: action
+          description: |
+              Required. A detailed rule defining how to route traffic.
```

Comment:
How is this required but also send_empty_value?

Reply:
Fixed